modules/dynamic-plugins/proc-extensions-installing.adoc

```diff
@@ -0,0 +1,52 @@
+[id="rhdh-extensions-plugins-installing_{context}"]
+= Installing a plugin by using Extensions
+You can install a plugin and configure it by updating the `{my-app-config-file}` file by using *Extensions*.
+
+== Prerequisites
+* You have the necessary permissions to modify plugin configurations and access the application environment.
+* You have identified and set the required environment variables referenced by the plugin's default configuration. These environment variables must be defined in the Helm Chart or Operator configuration.
+
+.Procedure
+. Use the search bar in the header to search for the plugin that you wish to install and click on the card. For example, search for Tekton and click *Read more* on the *Pipelines With Tekton* card.
++
+image::rhdh-plugins-reference/rhdh-extensions-tekton-card.png[Extensions catalog with Tekton card]
+. In the plugin drawer, you can review information about the plugin and how to configure it in {product-very-short}. To install the plugin click *Install*.
++
+image::rhdh-plugins-reference/rhdh-extensions-tekton-details.png[Extensions catalog with Tekton details]
+. On the install plugin page, a YAML editor and installation instructions are displayed.
++
+image::rhdh-plugins-reference/rhdh-extensions-tekton-editor-1.png[Extensions catalog with plugin editor template]
+. Click the *About the plugin* tab to view installation and configuration details for the plugin.
+. Click the *Examples* tab to display the default plugin configuration.
+. Click *Apply* to copy the default plugin configuration to the YAML editor.
+. In the YAML editor, click the copy icon to copy the plugin configuration.
++
+image::rhdh-plugins-reference/rhdh-extensions-tekton-editor-2.png[Extensions catalog with Tekton configuration]
++
+[NOTE]
+In {product-very-short} 1.6, the *Install* button is disabled, so you must copy the plugin configuration to the `{my-app-config-file}` file.
+. In the `{my-app-config-file}` file, add the plugin configuration that you copied in the previous step to the `plugins` definitions.
```

Comment:
The doc only mentions the configuration through `app-config.yaml`, but can be also configured by `dynamic-plugins.yaml`. 
On the other hand, the doc on disabling extension uses only `dynamic-plugins.yaml`